src/pages/javascript/check-the-length/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Check the Length Property of a String Variable
+title: Check the Length 
```

Comment:
I think this title is clearer with `of a string` added to it, as this doesn't talk about checking the length of an array.